make build fails for github.com/versent/saml2aws

**What happened:**
[A clear and concise description of what the bug is.]

Running `make build` fails with below error

```
make build
CGO_ENABLED=0 go build -ldflags "-X github.com/fidelity/kconnect/internal/version.commitHash=8d9d146 -X github.com/fidelity/kconnect/internal/version.buildDate=2020-10-07T15:24:06Z" .
go: github.com/versent/saml2aws@v1.8.5-0.20200622110128-d94772688a70: parsing third_party/saml2aws/go.mod: open /Users/arghyasadhu/go/src/github.com/arghya88/kconnect/third_party/saml2aws/go.mod: no such file or directory
make: *** [build] Error 1
```


**What did you expect to happen:**

Make build to succeed

**How to reproduce it:**

Execute `make build` 
**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**
Mac OSX
- kconnect version (use `kconnect version`):
- Kubernetes version (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):
- Target environment (e.g. EKS, AKS, Rancher):
- Authentication Used (e.g. SAML, IAM, Azure AD):

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -60,14 +60,20 @@ make test
 make lint
 ```
 
-7. Build and manually test kconnect locally:
+7. Use `git submodule` for saml2aws third party dependency
+
+```bash
+git submodule update --init --recursive
+```
+
+8. Build and manually test kconnect locally:
 
 ```bash
 make build
 ```
 
-8. Commit and push your branch. We follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for the commits and PRs. If the PR is a work in progress ensure that you prefix the title with WIP:.
-9. Create a pull request
-10. Check that the PR checks pass
+9. Commit and push your branch. We follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for the commits and PRs. If the PR is a work in progress ensure that you prefix the title with WIP:.
+10. Create a pull request
+11. Check that the PR checks pass
 
 Once a PR has been created it will be reviwed by one of the maintainers of kconnect.

